content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -41,6 +41,11 @@ and then further customize the configuration for your specific needs.
 
 1. [Install a Helm client](https://github.com/helm/helm/blob/master/docs/install.md) with a version higher than 2.10.
 
+1. For Istio with SDS enabled:
+  * Please use Kubernetes v1.12 or above.
+  * For non-GKE environments, please add [extra configurations](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/?origin_team=T382U9E4U#service-account-token-volume-projection) to your Kubernetes. You may
```

Comment:
Does this mean I have the restart the apiserver?

Reply:
For upgrading from an existing cluster, I think so. But this is for the installation Istio page, so I assume no Istio control plane and Istio-controlled workloads have been deployed, so it should not matter. For an existing cluster, they will need to upgrade apiserver and Citadel agent (to use trustworthy jwt, since the default used to be legacy jwt).